example/lib/main.dart

```diff
@@ -5,15 +5,80 @@ import 'package:flutter/services.dart';
 import 'package:sentry_flutter/sentry_flutter.dart';
 
 void main() {
-  runApp(MyApp());
+  SentryFlutter.init((o) {
+    // NOTE: Replace the DSN below with your own from your project settings in Sentry.
+    o.dsn =
```

Comment:
Even though this is the more "sentry" way, in the Dart/Flutter world, named parameter is likely a better choice here.
Parameters would need to be inverted then.